src/libtiled/maprenderer.cpp

```diff
@@ -106,10 +106,11 @@ static bool hasOpenGLEngine(const QPainter *painter)
             type == QPaintEngine::OpenGL2);
 }
 
-CellRenderer::CellRenderer(QPainter *painter)
+CellRenderer::CellRenderer(QPainter *painter, const bool isHexagonal)
```

Comment:
Please make this `isHexagonal` into an enum like `CellType` with default value `OrthogonalCells` and alternative value `HexagonalCells`.